src/foam/nanos/auth/twofactor/authy/AuthyService.js

```diff
@@ -0,0 +1,86 @@
+foam.CLASS({
+  package: 'foam.nanos.auth.twofactor.authy',
+  name: 'AuthyService',
+
+  documentation: 'Implementation of 2FA using Authy',
+
+  implements: [
+    'foam.nanos.NanoService'
+  ],
+
+  javaImports: [
+    'com.authy.AuthyApiClient',
+    'com.authy.api.Params',
+    'com.authy.api.PhoneVerification',
+    'com.authy.api.Verification'
+  ],
+
+  properties: [
+    {
+      class: 'Object',
+      name: 'client',
+      javaType: 'com.authy.AuthyApiClient',
+      hidden: true
+    },
+    {
+      class: 'Boolean',
+      name: 'debug'
+    },
+    {
+      class: 'String',
+      name: 'apiHost'
+    },
+    {
+      class: 'String',
+      name: 'apiKey'
+    }
+  ],
+
+  methods: [
+    {
+      name: 'sendToken',
+      javaReturns: 'boolean',
+      args: [
+        {
+          name: 'user',
+          javaType: 'foam.nanos.auth.User'
+        }
+      ],
+      javaCode:
+`AuthyApiClient client = getClient();
+if ( client == null )
+  return false;
+
+PhoneVerification phoneVerification = client.getPhoneVerification();
+Verification verification = phoneVerification.start(user.getPhone(), "1", "sms", new Params());
```

Comment:
@kgrgreer How would you suggest getting the user's country code for their phone number?